package.json

```diff
@@ -13,6 +13,17 @@
     "android": "react-native run-android",
     "snyk-protect": "snyk protect"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "jest ./__tests__/*",
+      "eslint ."
+    ]
+  },
   "rnpm": {
```

Comment:
Since you're here, can you check if we still need this?
I don't think so.

Reply:
deprecated, removed.